ethkey/src/secret.rs

```diff
@@ -55,6 +55,12 @@ impl FromStr for Secret {
 	}
 }
 
+impl From<&'static str> for Secret {
```

Comment:
`"0x....".into()` is substantially less visual clutter than `"0x.....".parse().unwrap()`.

static strings are a notable exception for "no panickers" rule since they, by and large, introduce no dynamic failure mode. in tests, they're certainly fine and just help rid the codebase of semantically useless noise in our test logic.